### ISSUE TITLE
There is no password change functionality

### ISSUE THREAD

**Issue by irshadpc** [cyclefortytwo/ironbelly#17]
There is no way to change the password, right now we need to destroy and wallet and backup with seeds.

**Comment by akatasonov** [comment]
isn't that how crypto wallets are expected to behave?

**Comment by i1skn** [comment]
@akatasonov we can optimise this process for the user. Essentially we do not need to destroy the whole wallet, but just re-encrypt the seed with the new password. Not sure about urgency of this feature, but I can see why people want to change their password from time to time.

**Comment by akatasonov** [comment]
@i1skn yeah, that's certainly possible. However, there should be lower level grin lib support for this feature to work. I don't see it yet?

**Comment by i1skn** [comment]
Yeah, my initial guess would be to implement it in grin first and then just use from Ironbelly.
